src/main/java/org/commcare/formplayer/exceptions/MultipleFormDefsFoundException.java

```diff
@@ -0,0 +1,8 @@
+package org.commcare.formplayer.exceptions;
+
+public class MultipleFormDefsFoundException extends RuntimeException {
```

Comment:
this is now unused

Reply:
See c231fefd5255161f63c4133f825ac4a03ff516ad